internal/devices/create.go

```diff
@@ -125,8 +125,13 @@ func (c *Client) Create() *cobra.Command {
 				if billingCycle != "" {
 					facilityDeviceRequest.DeviceCreateInFacilityInput.SetBillingCycle(*validBillingCycle)
 				}
-				if alwaysPXE {
-					facilityDeviceRequest.DeviceCreateInFacilityInput.SetAlwaysPxe(alwaysPXE)
+				if alwaysPXE != "" {
```

Comment:
These changes were addressed in https://github.com/equinix/metal-cli/pull/282#pullrequestreview-1550002522